docs/source/tutorial/function.rst

```diff
@@ -398,6 +398,40 @@ There are more functionalities on user-defined kernels in CuPy.
 :ref:`See the CuPy documentation on user-defined kernels for more details. <udkernel>`
 
 
+Write a function with training/test mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We sometimes want to make a function behave differently in training and test modes.
+The training/test mode in Chainer is configured by :data:`chainer.config`.
+This is a thread-local configuration object, and users can substitute True or False to its ``train`` attribute.
+You can refer to :ref:`configuration` to see how to configure this flag as well as other configuration items.
+
+Here, we just show how to use this flag to make a function support training/test mode.
+What you have to do is just checking ``chainer.config.train`` and branches the procedure.
```

Comment:
I suggest something like one the following:

What you have to do is just check ``chainer.config.train`` and branch the procedure.

or

You will need to check the value of the boolean flag ``chainer.config.train`` and branch appropriately.